lib/ruby_lsp/requests/folding_ranges.rb

```diff
@@ -77,7 +77,8 @@ def visit(node)
 
         case node
         when *SIMPLE_FOLDABLES
-          add_node_range(T.must(node))
+          location = T.must(node).location
+          add_lines_range(location.start_line, location.end_line - 1)
```

Comment:
Can we encapsulate this logic inside add_lines_range method. From what I can see, it always takes a start and and end line number and the end is always 1 less than the end of the location. We could instead always send a location object into the method and it can be responsible for subtracting 1 from the end line numbers. It would be better for encapsulation.

Reply:
Ah ok, makes sense.